### DISCUSSION TITLE
치지직 중간광고 재생시 검은화면

### DISCUSSION THREAD

**Discussion by natic0309** [List-KR/community#67] [General]
안녕하세요 치지직 중간광고가 나올시 화면에 검은색 팝업처럼 화면이 가려지는현상이있습니다 혹시 수정이 가능할까요?

**Comment by piquark6046** [comment]
중간 광고는 언제 일어나나요?

**Comment by natic0309** [comment]
중간광고는 스트리머 설정마다 다른데 30분 or 45분 간격으로 등장합니다

**Reply by piquark6046** [reply]
현재 사용하시고 계신 애드블록의 종류가 AdGuard인가요? uBlock Origin인가요?

**Comment by natic0309** [comment]
AdGuard 윈도우버전입니다

**Reply by piquark6046** [reply]
Naver 그리드 컴퓨팅 프로그램이 설치되어 있나요?
안랩에서 배포하는 안티-바이러스나 금융 보안 프로그램이 컴퓨터에 설치되어 있나요?
라이선스 데이터를 제외한 AdGuard for Windows의 설정 파일을 내보셔서 여기 또는 filters@adguard.com 으로 보내주시겠어요?

**Reply by piquark6046** [reply]
또한, 사용하시는 브라우저의 이름은 무엇인가요?

**Reply by natic0309** [reply]
네이버 그리드 프로그램은 설치되어있고 안랩관련 프로그램은 설치되어있지 않습니다 브라우저는 엣지 브라우저 사용중이며
ad가드에서 설정내보내기 눌러서 해당이메일로 보내드리면되는건가요? 라이센스 데이터제외하는방법도 알려주실수있으실까요?

**Reply by piquark6046** [reply]
네. 보내주시면 됩니다. 그리고 AdGuard for Windows에서는 설정 파일에 라이선스 데이터가 없다고 하네요.

**Reply by natic0309** [reply]
넵 보냈습니다

**Reply by piquark6046** [reply]
@natic0309 이메일을 확인해주세요.

**Reply by natic0309** [reply]
네네 로그인시청중 나타난 현상입니다
